docs/users/security/overview.md

```diff
@@ -0,0 +1,34 @@
+<!--
+order: 1
+-->
+
+# Overview
+
+Learn about security on Evmos {synopsis}
+
+The [Security Policy on Evmos](https://github.com/evmos/evmos/security/policy)
+includes a vulnerability disclosure policy and guidelines for
+reporting vulnerabilities in the Evmos platform.
+It explains the process for reporting and disclosing vulnerabilities through
+the use of the [Immunefi Bug Bounty Program](https://immunefi.com/bounty/evmos/) and <!-- markdown-link-check-disable-line -->
+how the Evmos team handles and resolves reported vulnerabilities.
+The policy includes guidelines for researchers to follow
+when reporting vulnerabilities, such as avoiding privacy violations
+and keeping information confidential, as well as commitments
+from the Evmos team, such as not pursuing legal action
+and working with researchers to disclose issues in a timely manner.
```

Comment:
I think this first section is too repetitive, the first 3 sentences say "report[ing/ed] vulnerabilities" 3 times and the word vulnerability is repeated 5 times

```suggestion
The contents of the policy help avoiding privacy violations
and keeping information confidential.
Also included are commitments from the Evmos team,
such as not pursuing legal action
and working with researchers to disclose issues in a timely manner.
```